buildpack.md

```diff
@@ -991,3 +993,19 @@ The stack ID:
 A buildpack descriptor that specifies `order` MUST be [resolvable](#order-resolution) into an ordering of buildpacks that implement the [Buildpack Interface](#buildpack-interface).
 
 A buildpack reference inside of a `group` MUST contain an `id` and `version`.
+
+### io.buildpacks.app.source OCI Image label
+The value of this label:
+- MUST be a string of escaped json complying with [RFC 8259](https://tools.ietf.org/html/rfc8259).
+- when unescaped, MUST comply with the following schema:
+```json
+{
+  "$schema": "http://json-schema.org/schema#",
```

Comment:
I like the idea of using json schema although it's not currently a standard. Not opposed to it but wanted to raise it up as something that we should maybe align on and add to the other parts of the spec.

Reply:
We've removed this section at present at the suggestion of @sclevine (see comments below).
We'd like to see JSON schema being used in the future.